samples/bootloader/README.rst

```diff
@@ -183,6 +184,12 @@ When using the ``nrf52840dk_nrf52840`` board, this would produce a flash memory
 
    B0 flash memory layout with MCUboot
 
+.. note::
+   When using DFU, and especially for production-ready devices, these parititions must be configured in the following way to meet :ref:`ug_bootloader_flash_static_requirement`:
+
+   * *B0*, *S0*, *S1* - These slots must be statically partitioned.
+   * *mcuboot_primary* and *mcuboot_secondary* - These slots can be dymanically partitioned.
```

Comment:
This is a bit tricky, because we still must ensure compatibility of the application with partitions known to MCUboot bootloader

Reply:
How about 
`These slots can be dymanically partitioned. Keep in mind that MCUboot and application knowledge of partitions must match.` ?

Reply:
Moved to configuration.